### PR TITLE
OG Tags - DG

### PR DESCRIPTION
**Pull request by tolgaouz**
- Created a variable called image which the SEO plugin will inherit the value for og:image tag.
- We now have 3 different variables pointing to images:
    - cover ( Image that is shown in thumbnail )
    - header ( Image that is shown in the beginning of the post view )
    - image ( OG:Image )

### CODE REVIEW

**Comment by davidgatti** [comment]
Screen Shot 2020-07-17 at 9 04 25 AM

The type is set, twice, you need to figure out how to use the module to use the Type the right way. Please be more carfeull with your work, double check your chagnes.

File: _includes/head.html [hunk 1]
```diff
@@ -12,6 +12,11 @@
         <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <!-- Adding meta-tags that are not added by seo plugin -->
+    <meta property="og:type" content="website"/>
```
Comment (davidgatti) [inline _includes/head.html:17]: Double tags

**Comment by davidgatti** [comment]
@tolgaouz check the conflict.

**Comment by tolgaouz** [comment]
Resolved the conflict. Closing

**Pull request merged**